Tests/PostgresKitTests/PostgresKitTests.swift

```diff
@@ -15,7 +15,7 @@ class PostgresKitTests: XCTestCase {
     }
     
     func testPerformance() throws {
-        let db = PostgresConnectionSource(configuration: .test)
+        let db = PostgresConnectionSource(sqlConfiguration: .test)
```

Comment:
Do we have any deprecated APIs for the old one? I couldn't spot them

Reply:
Yep, it's right here: https://github.com/vapor/postgres-kit/pull/241/files/995aa613ff3a3fb9196d666f31e2bdee3ad0adc3#diff-5c462d96b4d820bf87f30fef51a05da029789e0fb4359d1a9fed1793518b3dc4R9